.github/pull_request_template.md

```
@@ -1,23 +1,16 @@
 <!--- Make use of markdown lists. They make stuff much easier to read through. -->
-### Purpose
+### Purpose and Description
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)